x/profiles/keeper/keeper.go

```diff
@@ -135,8 +133,8 @@ func (k Keeper) RemoveProfile(ctx sdk.Context, address string) error {
 	}
 
 	if !found {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest,
-			"no profile associated with the following address found: %s", address)
+		return sdkerrors.Wrapf(types.ErrProfileNotFound,
+			"no profile associated with the following address: %s", address)
```

Comment:
I think it's sufficient to simply let `Wrap` do the wrapping here
```suggestion
		return sdkerrors.Wrap(types.ErrProfileNotFound, address)
```